README.org

```diff
@@ -7,8 +7,34 @@ s-expressions.  S-expressions are defined by the following type:
 type sexp = Atom of string | List of sexp list
 #+end_src
 
-and are rendered as parenthesized lists of strings, e.g., =(This (is
-an) (s expression))=.
+
+** Usage example
+
+In this example we build an Sexp expression that corresponds to
+=(This (is an) (s expression))=, serialize a Sexp expression into a string,
+and then parse a string into an Sexp.
+
+#+begin_src ocaml
+open Sexplib
+open Sexplib.Sexp
+
+let () =
+  (* Build an Sexp from: (This (is an) (s expression)) *)
+  let exp1 = List [
+    Atom "This";
+    List [Atom "is"; Atom "an"];
+    List [Atom "s"; Atom "expression"]
+  ] in
+  (* Serialize an Sexp object into a string *)
+  print_endline(Sexp.to_string exp1);
```

Comment:
Please add a space between print_endline and its argument, and after the assert below.

Reply:
I've pushed a --force commit that resolves this issue.